spec/Script.md

```diff
@@ -1,8 +1,8 @@
-Codechain saves the unlock condition for each asset, and anyone who can pass this condition is authorized to use the asset. The unlock condition is represented in a byte array, and decoded as a list of instructions before execution. Script language is intentionally designed to not be Turing-complete, and all scripts are ensured to be finished in finite time.
+Codechain saves the unlock condition for each address, and anyone who can pass this condition is authorized to use the address. The unlock condition is represented in a byte array, and decoded as a list of instructions before execution. Script language is intentionally designed to not be Turing-complete, and all scripts are ensured to be finished in finite time.
```

Comment:
Ditto. `address` is not an appropriate substitute for `asset`

Reply:
I modified it. : D